### ISSUE TITLE
Dataset + training script for re-training the EMOCA controlnet?

### ISSUE THREAD

**Issue by nlml** [foivospar/Arc2Face#30]
Thanks for this awesome work.

I am looking through the paper and code, but can't seem to find many details on exactly how you trained the EMOCAv2-driven controlnet?

Would you potentially be able to post a training script or some details about how exactly this model was trained? Also was it trained on the same dataset as arc2face, and how long was it trained for?

Thanks again!

**Comment by arc2face** [comment]
Hello,

ControlNet's training was done based on the official example [script](https://github.com/huggingface/diffusers/blob/main/examples/controlnet/train_controlnet.py) provided by diffusers with minor adjustments (e.g. using our ArcFace conditioning mechanism and our pre-trained checkpoints). 

We used only FFHQ to train the ControlNet. You should get good results relatively early. In our case, we trained it for 50 epochs with a learning rate of 1e-5. 

To generate the conditioning EMOCAv2 normal maps of the FFHQ images, you can follow our code [here](https://github.com/foivospar/Arc2Face/blob/723aec08257c6c7a95020a5542544c6ca80c0166/gradio_demo/app_controlnet.py#L112).

**Comment by foivospar** [comment]
Hello,

ControlNet's training was done based on the official example [script](https://github.com/huggingface/diffusers/blob/main/examples/controlnet/train_controlnet.py) provided by diffusers with minor adjustments (e.g. using our ArcFace conditioning mechanism and our pre-trained checkpoints).

We used only FFHQ to train the ControlNet. You should get good results relatively early. In our case, we trained it for 50 epochs with a learning rate of 1e-5.

To generate the conditioning EMOCAv2 normal maps of the FFHQ images, you can follow our code [here](https://github.com/foivospar/Arc2Face/blob/723aec08257c6c7a95020a5542544c6ca80c0166/gradio_demo/app_controlnet.py#L112).

**Comment by nlml** [comment]
Thank you very much!
